arrocke/gloss-translation
# refactor: use passwords for auth

## What has changed


* Added passwords to accept invites and login routes and views
* Added length and confirmation validators to the text input
* Modified the error message component to be more flexible. When you use it, you have to specify each error message.
* Pulled out the users translation strings into a separate json file

## Connected Issues



relates to #124 

## QA Steps



1. Add the following value to `hashed_password` in your admin user in the database:
`s2:h2I35PXwsp81qZNX:fcb5e8e8bbb2b6bd93cad7d4af84c8057f58254d45574a3809fb3455ca88a7ee6d4690c89ea6c63e66bda1f5229da8b95f4227e5a6e21cac6acd83dcd4b730c7`
2. Try logging in with password `asdf1234`
3. Invite a user and confirm that the password you used to sign up works to log in

## Post-Deployment



We'll need to add passwords to existing users
